sites/fast-website/src/public/index.html

```diff
@@ -52,7 +52,7 @@ <h1 class="section-heading-hero">The adaptive interface system for modern web ex
 
         <section class="fast-frame-section" id="adaptive-ui">
           <div class="section-content">
-            <site-section-header class="fast-frame-header">
+            <site-section-header class="section-header" class="fast-frame-header">
```

Comment:
Do we want 2 classes here? If we do it should be `class="section-header fast-frame-header"`

Reply:
Thanks for catching that Jeff.  My bad for doing a find and replace.  Fixed.